chore(instrumentation-graphql): turn off failing test and add CTA for becoming an owner (#1201)
open-telemetry/opentelemetry-js-contrib: plugins/node/opentelemetry-instrumentation-graphql/README.md

# OpenTelemetry Instrumentation GraphQL

> :bangbang: You could be a component owner for this package and maintain it to bring it up to quality it's users deserve! Check out [open issues](https://github.com/open-telemetry/opentelemetry-js-contrib/labels/pkg%3Ainstrumentation-graphql) on how to help.

Reviewer: ```suggestion
:bangbang: You could be a component owner for this package and help maintain the quality its users deserve! Check out [open issues](https://github.com/open-telemetry/opentelemetry-js-contrib/labels/pkg%3Ainstrumentation-graphql) on how to help.
```
